src/components/Practice/FluencyP5.jsx

```
@@ -818,226 +802,245 @@ const FluencyP5 = ({
-              zIndex: 11000,
+              width: isMobile ? "36px" : "50px",
+              height: isMobile ? "36px" : "50px",
+              position: "absolute",
+              top: isMobile ? "5px" : "20px",
+              left: isMobile ? "5px" : "20px",
+              cursor: "pointer",
+              zIndex: 1000,
             }}
-          >
+            onClick={() => setOpen(true)}
+          />
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)